workflow/full-installertest-flow.groovy

```diff
@@ -0,0 +1,66 @@
+// Full installer test flow, in one file
+// You must parameterize the build with: 
+
+// @stringparameter dockerLabel (node label for docker nodes)
+// @stringparameter packagingBranch - branch to use in packaging build
+
+// **ARTIFACTS URLS**
+// Note: you can use an artifact archived in a Job build by an artifact:// URL
+//  Ex: 'artifact://full/path/to/job/buildNr#artifact.ext'
+// @stringparameter debfile URL to Debian package
+// @stringparameter rpmfile URL to CentOS/RHEL RPM package
+// @stringparameter susefile URL to SUSE RPM package
+// @stringparameter testLibraryBranch branchname to load the installertests workflow lib from
+
+// Optional build parameters
+// @stringparameter (optional) port - port number to use in testing jenkins (default 8080)
+// @stringparameter (optional) artifact (jenkins artifactname, defaults to 'jenkins')
+
+// Basic parameters
+String dockerLabel = 'ubuntu-ope01'
```

Comment:
Better solution:  default to master (once merged), unless you override to use another branch.  Renaming the variable anyway since the existing name is kind of unhelpful.